### PR TITLE
Add statosio w/ git auto-update

### PR DESCRIPTION
**Pull request by a6b8**
Statosio.js is an add-on function for d3.js to generate simple chart diagrams, in a fast and reliable way.

https://d3.statosio.com

### CODE REVIEW

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
Please fix the issues the CI is raising

**Comment by a6b8** [comment]
> Please fix the issues the CI is raising

Error was: "no tagged version found in git"

I created a release with a version tag. should working now.

File: packages/s/statosio.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "statosio.js",
```
Comment (MattIPv4) [inline packages/s/statosio.json:2]:
```suggestion
  "name": "statosio",
```
Match filename

File: packages/s/statosio.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        ]
+      }
+    ]
+  },
+  "license": "MIT",
+  "authors": [
+    {
+        "name": "Andreas Banholzer",
+        "email": "hello@13plus4.com",
+        "url": "13plus4.com"
```
Comment (MattIPv4) [inline packages/s/statosio.json:37]:
```suggestion
      "name": "Andreas Banholzer",
      "email": "hello@13plus4.com",
      "url": "13plus4.com"
```
Correct 2-space indentation

File: packages/s/statosio.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    "bar-stacked-chart"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/a6b8/statosio.js.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```
Comment (MattIPv4) [inline packages/s/statosio.json:27]:
```suggestion
          "*.js"
```
Open wildcards are dangerous, let's restrict this

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

**Review by MattIPv4 — APPROVED** [review]
Lgtm, ty

**Pull request merged**